src/region.js

```diff
@@ -58,6 +59,8 @@ const Region = MarionetteObject.extend({
 
     if (view === this.currentView) { return this; }
 
+    this.swappingView = !!this.currentView;
```

Comment:
For consistency we should use an `_is` prefixed variable for state flags with an `is____()` accessor.